packages/components/nodes/embeddings/GoogleVertexAIEmbedding/GoogleVertexAIEmbedding.ts

```diff
@@ -33,11 +33,35 @@ class GoogleVertexAIEmbedding_Embeddings implements INode {
             description:
                 'Google Vertex AI credential. If you are using a GCP service like Cloud Run, or if you have installed default credentials on your local machine, you do not need to set this credential.'
         }
-        this.inputs = []
+        this.inputs = [
+            {
+                label: 'Model Name',
+                name: 'modelName',
+                type: 'options',
+                options: [
+                    {
+                        label: 'textembedding-gecko@001',
+                        name: 'textembedding-gecko@001'
+                    },
+                    {
+                        label: 'textembedding-gecko@latest',
+                        name: 'textembedding-gecko@latest'
+                    },
+                    {
+                        label: 'textembedding-gecko-multilingual@latest',
+                        name: 'textembedding-gecko-multilingual@latest'
+                    }
+                ],
+                default: 'textembedding-gecko@001',
+                optional: true,
+                additionalParams: true
```

Comment:
maybe we don't have to use `additionalParams` here

Reply:
Got it！